guvnor-rest/guvnor-rest-backend/src/main/java/org/guvnor/rest/backend/ProjectResource.java

```diff
@@ -279,7 +279,7 @@ public Response createProject(
             throw new WebApplicationException( Response.status( Response.Status.NOT_FOUND ).entity( repositoryName ).build() );
         }
 
-        Set<Project> projects = projectService.getProjects(repository, repository.getCurrentBranch());
+        Set<Project> projects = projectService.getProjects(repository, "master");
```

Comment:
I thought you were trying to remove use of ```master```? I also assume enforces that ```REST``` operations work only on ```master```?